client/landing/gutenboarding/components/domain-picker/index.tsx

```diff
@@ -93,87 +104,95 @@ const DomainPicker: FunctionComponent< Props > = ( {
+						<div className="domain-picker__suggestion-header-title">
+							{ NO__( 'Professional Domain' ) }
+						</div>
+						<div className="domain-picker__suggestion-header-description">
+							{ __experimentalCreateInterpolateElement(
+								NO__( '<Price>Free</Price> for the first year with a paid plan' ),
+								{ Price: <em /> }
+							) }
+						</div>
+					</div>
+					<div className="domain-picker__suggestion-item-group">
+						{ paidSuggestions?.length
+							? paidSuggestions.map( suggestion => (
+									<DomainPickerSuggestionItem
+										suggestion={ suggestion }
+										isRecommended={ suggestion === recommendedSuggestion }
+										isCurrent={ currentDomain.domain_name === suggestion.domain_name }
+										onClick={ () => onDomainPurchase( suggestion ) }
+										key={ suggestion.domain_name }
+									/>
+							  ) )
+							: times( paidPlaceholderAmount, i => (
+									<DomainPickerSuggestionItemPlaceholder key={ i } />
+							  ) ) }
+					</div>
+				</PanelRow>
 
 				<PanelRow className="domain-picker__panel-row">
-					<div className="domain-picker__recommended-header">{ NO__( 'Recommended' ) }</div>
-					{ suggestions?.length
-						? suggestions.map( suggestion => (
-								<Button
-									onClick={ () => handleDomainClick( suggestion ) }
-									className="domain-picker__suggestion-item"
-									key={ suggestion.domain_name }
-								>
-									<span className="domain-picker__suggestion-item-name">
-										{ suggestion.domain_name }
-									</span>
-									<span className="domain-picker__suggestion-action">
-										{ suggestion.is_free ? (
-											<span>
-												<span className="domain-picker__price">{ NO__( 'Free' ) }</span>
-												{ NO__( 'Select' ) }
-											</span>
-										) : (
-											<span>
-												{ /* FIXME: What value do we show here? */ }
-												<span className="domain-picker__price">{ NO__( 'from €4/month' ) }</span>
-												{ NO__( 'Upgrade' ) }
-											</span>
-										) }
-									</span>
-								</Button>
-						  ) )
-						: times( placeholderAmount, i => (
-								<Button className="domain-picker__suggestion-item" key={ i }>
-									<span className="domain-picker__suggestion-item-name placeholder">
-										example.wordpress.com
-									</span>
-									<span className="domain-picker__suggestion-action placeholder">
-										{ NO__( 'Select' ) }
-									</span>
-								</Button>
-						  ) ) }
+					<div className="domain-picker__suggestion-header">
+						<div className="domain-picker__suggestion-header-title">{ NO__( 'Subdomain' ) }</div>
+					</div>
+					<div className="domain-picker__suggestion-item-group">
+						{ freeSuggestions?.length
+							? freeSuggestions.map( suggestion => (
+									<DomainPickerSuggestionItem
+										suggestion={ suggestion }
+										isCurrent={ currentDomain.domain_name === suggestion.domain_name }
```

Comment:
This is a type error, isn't it?
```suggestion
										isCurrent={ currentDomain?.domain_name === suggestion.domain_name }
```